strawberryfields/backends/states.py

```diff
@@ -1266,26 +1309,97 @@ def parity_expectation(self, modes):
 
         return parity
 
-    @abc.abstractmethod
+    def ket(self, **kwargs):
+        cutoff = kwargs.get("cutoff", 10)
+        mu = self._mu
+        cov = self._cov
+
+        if self._pure:
+            return twq.state_vector(
+                mu, cov, hbar=self._hbar, normalize=True, cutoff=cutoff, check_purity=False
```

Comment:
Just curious for my own understanding: how come `normalize=True` is needed here?

Reply:
Formally, you are only guaranteed a normalized state if the `cutoff` -> infinity. Typically if the `cutoff` is sufficiently large one is very close to being normalized.